app/src/main/res/layout/fragment_fxa_login.xml

```diff
@@ -14,21 +14,35 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         tools:context=".view.FxALoginFragment">
-    <include layout="@layout/include_backable" />
+
+    <include layout="@layout/include_backable"
+             app:layout_constraintVertical_chainStyle="packed"/>
+    <include layout="@layout/fragment_warning"
+             android:layout_width="match_parent"
+             android:layout_height="@dimen/hiddenNetworkError"
+             android:visibility="gone"
+             app:layout_constraintTop_toBottomOf="@id/toolbar"
+             app:layout_constraintBottom_toTopOf="@id/webView"
+             tools:text="@string/no_internet_connection"
+             app:layout_constraintVertical_chainStyle="packed"/>
 
     <WebView
             android:id="@+id/webView"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_marginTop="?attr/actionBarSize"
-            android:layout_weight="1"/>
+            app:layout_constraintVertical_chainStyle="packed"
+            app:layout_constraintTop_toBottomOf="@id/toolbar"
+            app:layout_constraintBottom_toTopOf="@id/skipFxA"
```

Comment:
This could be very risky here, when the `skipFxA` button is not visible. e.g. in release builds.

Reply:
good point. I'll change that around